package.json

```diff
@@ -69,6 +70,7 @@
     "clean": "yarn lerna run --no-bail clean",
     "check-dependencies": "node ./scripts/check-mismatched-dependencies.cjs",
     "docs": "typedoc --tsconfig tsconfig.build.json",
+    "typedoc-markdown": "typedoc --plugin typedoc-plugin-markdown",
```

Comment:
same questions as in the other PR.

plus: why doesn't it need the `tsconfig.build.json` ?

Reply:
Good call! Updating both PRs to use `tsconfig.build.json` now...